### PR TITLE
Added new ansible role for galaxy

### PR DESCRIPTION
**Pull request by s2504s**
## What
* Added ansible role for deploy datadog's agent on servers with support for CentOS, Debian, and Ubuntu 

## Why
* Needs to be public this role on Ansible Galaxy (galaxy.ansible.com)

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -1,16 +1,34 @@
 ---
```
Comment (const-bon) [inline meta/main.yml]: Revert these changes

```
 -
 -dependencies: [ ]
 -
  galaxy_info:
 -  author: David Reed
 -  description: Datadog agent installation
 +  author: 'Cloudposse'
 +  description: Datadog agent and modules installation
    license: MIT
 -  min_ansible_version: 2.0
 +  min_ansible_version: 1.6
```

File: defaults/main.yml [hunk 2]
```diff
@@ -1,11 +1,2 @@
 ---
```
Comment (const-bon) [inline defaults/main.yml:1]: Revert these changes:

```
 -datadog:
 -  api_key: changeme
 -  autostart_agent: yes
 -  dd_url: https://app.datadoghq.com
 -  default_tags: [ ]
 -  enabled: yes
 -  integrations: { }
 -  socket: "/opt/datadog-agent/run/datadog-supervisor.sock"
 -  tags: [ ]
```

**Comment by const-bon** [comment]
Revert README.md

**Comment by const-bon** [comment]
I think we can adjust our changes in a way we can make pull request to main repository.
What do you think, @s2504s ?

File: defaults/main.yml (hunk 2 above)
Comment (osterman) [reply defaults/main.yml:1]: We can keep these as-is. I'm okay with diverging with upstream.

File: meta/main.yml (hunk 1 above)
Comment (osterman) [reply meta/main.yml]: Let's revert these so we maintain attribution.

**Comment by s2504s** [comment]
A license version in meta has been fixed

**Comment by const-bon** [comment]
I think we can skip reverting `defaults/main.yml` and `README.md` for now, as we're not going to make PR to main (origin) repository.

**Review by osterman — APPROVED** [review]
